src/leapipc/IPCFileEndpoint.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2012-2016 Leap Motion, Inc. All rights reserved.
```

Comment:
Oh yeah we need to fix `scripts/copyright_check.sh`. Will probably do that in LeapIPC 0.1.4...